src/elements/Rail/Rail.js

```diff
@@ -0,0 +1,67 @@
+import _ from 'lodash'
+import React, { PropTypes } from 'react'
+import cx from 'classnames'
+
+import * as sui from '../../utils/semanticUtils'
+import { getUnhandledProps, useKeyOnly, useKeyOrValueAndKey } from '../../utils/propUtils'
+import META from '../../utils/Meta'
+
+function Rail(props) {
```

Comment:
The main description in the docs for a component comes from the doc block on the component itself.  Let's add this from the SUI docs:

```js
/**
 * A rail is used to show accompanying content outside the boundaries of the main view of a site
 */
function Rail(props) {
  // ...
}